### DISCUSSION TITLE
I want sliding slices to be silky smooth like Mayavi and rendered in real time!

### DISCUSSION THREAD

**Discussion by ChenglongLi777** [pyvista/pyvista#6091] [Ideas]
### Describe the feature you would like to be added.

I have an idea for a new feature!
I want sliding slices to be silky smooth like Mayavi and rendered in real time!

### Pseudocode or Screenshots

```python
Insert code here
```
Or drag your screenshot here!

**Comment by tkoyama010** [comment]
Could you give us a link to the URL of the Mayavi document you are discussing?
